ui/css/utilities/colors.scss

```diff
@@ -9,6 +9,10 @@ Before adding a color here make sure that there isn't a design token available.
   --mainnet: #29b6af;
   --inherit: inherit;
   --transparent: transparent;
+  --color-network-goerli-default: #1098fc;
+  --color-network-sepolia-default: #cfb5f0;
+  --color-network-goerli-inverse: #fcfcfc;
+  --color-network-sepolia-inverse: #fcfcfc;
```

Comment:
These network colors are slated for removal from v4 of the design tokens but are still used in the codebase. We are moving them to the extension stylesheet as they are third-party colors and will not change. We should deprecate third-party network colors in favor of using images instead.

Reply:
Moved to this PR https://github.com/MetaMask/metamask-extension/pull/24953/files#diff-e94da7b5420af178ae8a2ae327e1dc681421106b5b02b5a1a5f88c959b2bcedb